Economy.md

```diff
@@ -0,0 +1,93 @@
+# EssentialsX Economy
+The built in economy API will redirect to any other detected economy system such as vault. This means you can use EssEco api as a economy api instead of vault if your plugin is dependent on Essentials.
+## Commands
```

Comment:
```suggestion

## Commands
```

Just a formatting thing, it's easier to read the source if there's spaces before each section heading. I'll add these so don't worry about this.